<!doctype html>
<!--[if lt IE 7]><html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="en"> <![endif]-->
<!--[if (IE 7)&!(IEMobile)]><html class="no-js lt-ie9 lt-ie8" lang="en"><![endif]-->
<!--[if (IE 8)&!(IEMobile)]><html class="no-js lt-ie9" lang="en"><![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en"><!--<![endif]-->
<head>
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  

  
    <title>Research</title>
    <meta name="description" content="Our Research">
    <link rel="canonical" href="https://thehamiltonlab.github.io/hamilton-lab/research/">
  

  <script>
    /* Cut the mustard */
    if ( 'querySelector' in document && 'addEventListener' in window ) {
      document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + 'js';
    }
  </script>

  <link rel="stylesheet" href="/hamilton-lab/assets/css/main.css">
  <!-- start custom head snippets -->

<!-- insert favicons. use http://realfavicongenerator.net/ -->

<!-- end custom head snippets -->

</head>

</head>

<body id="page">


  <div class="navigation-wrapper">
    <a href="#menu-toggle" id="menu-toggle">Menu</a>
    <nav id="primary-nav" class="site-nav animated drop">
      <ul><li><a href="/hamilton-lab/hamilton-lab/">The Hamilton Lab</a></li><li><a href="/hamilton-lab/research/">Research</a></li><li><a href="/hamilton-lab/people/">People</a></li><li><a href="https://scholar.google.com/citations?user=CKTkwMoAAAAJ&hl=en">Publications</a></li><li><a href="/hamilton-lab/prospective-students/">Prospective Students</a></li><li><a href="/hamilton-lab/contact/">Contact</a></li></ul>
    </nav>
  </div><!-- /.navigation-wrapper -->



<div id="main" role="main">
  <article class="entry">
    <img src="https://thehamiltonlab.github.io/images/wet_web.jpg" class="entry-feature-image" alt="Research" >
    <div class="entry-wrapper">
      <header class="entry-header">
        <h1 class="entry-title">Research</h1>
      </header>
      <div class="entry-content">
        <p><strong>“Nothing in evolution makes sense except when seen in the light of phylogeny.”</strong></p>

<p><em>- Jay Savage, evolutionary biologist</em></p>

<h3 id="overview"><strong>Overview</strong></h3>
<p>Understanding how Earth’s diversity has been shaped by evolution is one of the key objectives in biological research, and a compelling mechanism for engaging and educating the public. My research is driven by three major questions: 1) What are the relationships within the arthropod Tree of Life?; 2) How has evolution produced such an astonishing array of form and function?; and 3) Why are particular lineages more diverse than others?.</p>

<p><strong>Topics of interest to our lab group:</strong></p>
<ol>
  <li>Spider and Bombycoidea moth systematics
    <ul>
      <li>Build better understanding of Araneae &amp; Lepidoptera branches on the Tree of Life</li>
      <li>Train the next generation of taxonomists</li>
    </ul>
  </li>
  <li>The bat/moth evolutionary “arms race”</li>
  <li>Comparative genomics of phenotypic change
    <ul>
      <li>Body size</li>
      <li>Eye loss</li>
      <li>Wing shape</li>
    </ul>
  </li>
  <li>Body size evolution</li>
  <li>Convergent evolution</li>
  <li>Use of genomics as a tool in STEM outreach</li>
</ol>

<h3 id="current-and-future-projects"><strong>Current and Future Projects</strong></h3>
<h3 id="systematics">Systematics</h3>

<p>Our research addresses a wide range of questions by investigating the evolutionary history of the Araneae (from alpha and beta taxonomy to macroevolutionary dynamics to species delimitation), with a focus predominantly on the Mygalomorphae, and in particular the Theraphosidae (tarantulas). We also work on Lepidoptera, predominantly the superfamily Bombycoidea, and in particular the Saturniidae (silkmoths).</p>

<p>*Evolution and systematics of <i>Aphonopelma<i> -* <i>Aphonopelma<i> is a group where traditional morphological characters were generally ineffective at evaluating inter- and intraspecific variation, with the genus declared “one of the greatest known challenges to species delimitation in spiders”. The principal goal of this research was and is to formally resolve the species-level diversity by using an integrative approach that combined phylogenomics with morphological, geographic, and behavioral data to define species boundaries.</i></i></i></i></p>

<p><em>The Theraphosidae Tree of Life -</em> Future research will expand this systematic work throughout the family Theraphosidae (tarantulas), resolving relationships and attempting to explain how this family became one of the most diverse of all spiders.</p>

<p>*The evolution of eye-loss in a genus of troglophilic tarantulas, <i>Hemirrhagus<i> -* One important aspect of this work will be my growing collaboration with researchers in Mexico – an incredibly diverse region with a large number of species that have been overlooked, under-sampled, and undescribed. A major focus of this collaborative endeavor will be to better understand the evolution of the blind, troglophilic tarantula species of Mexico – the only place where this is known to occur.</i></i></p>

<figure class="pull-right">
	<a href="https://thehamiltonlab.github.io/images/mygalomorph_phylo.jpg">&lt;img src="https://thehamiltonlab.github.io/images/mygalomorph_phylo.jpg" alt="image" width="150"</a>
</figure>
<figure class="pull-right">
	<a href="https://thehamiltonlab.github.io/images/Fig1_moth_tree.jpg">&lt;img src="https://thehamiltonlab.github.io/images/Fig1_moth_tree.jpg" alt="image" width="150"</a>
</figure>

<h3 id="body-size-evolution">Body Size Evolution</h3>

<p>Body size is one of the most important determinants of an organism’s ecological role (Hanken and Wake 1993). Because it is correlated with physiological and fitness characters, body size has been one of the most important traits evolutionary biologists have investigated. My research explores three major stories of dramatic changes in body size:  1) Miniaturization in sympatric lineages of Aphonopelma tarantulas; 2) Sexual size dimorphism in the golden orb-weaving spiders (Nephilidae); and 3) The evolution of large body sizes as an anti-bat trait in the evolutionary “arms race” between bats and moths. Future work will utilize phylogenetics, morphometrics, and comparative genomics (de novo genome sequencing and transcriptomics) to investigate the evolutionary path of these traits, as well as the putative mechanisms and the genome regions that have provided for these changes.</p>

<figure class="pull-right">
	<a href="https://thehamiltonlab.github.io/images/anax_paloma_size.jpg"><img src="https://thehamiltonlab.github.io/images/anax_paloma_size.jpg" alt="image" width="150" /></a>
</figure>

<figure class="pull-right">
	<a href="https://thehamiltonlab.github.io/images/Sys_Bio_cover.jpg"><img src="https://thehamiltonlab.github.io/images/Sys_Bio_cover.png" alt="image" width="150" /></a>
</figure>

<h3 id="the-bat-moth-evolutionary-arms-race">The Bat-Moth Evolutionary Arms Race</h3>

<p>With an estimated 140,000 described species, an enormous array of wing shapes and body sizes exist across Lepidoptera. However, until recently few studies investigated the drivers of this spectacular morphological diversity. A major hypothesis I am working to answer is whether differences in moth wing shape and body size are associated with their primary nocturnal predators – bats, and if these trait differences relate to clade diversity. With their suite of anti-predator strategies (i.e., ears keen to a bat’s ultrasonic echolocation, acrobatic evasive flight, and specialized morphology to aid in defensive flight and escape mechanisms – e.g., hindwing tails and ultrasound-producing organs that “jam” bat biosonar), moths in the superfamily Bombycoidea are an ideal system to test evolutionary hypotheses concerning the historical path of anti-bat traits.
Future genomic research will continue gathering the data needed to: 1) test whether there are correlated changes between anti-bat strategies and increasing/decreasing rates of speciation or extinction; and 2) decipher the functional genomics of wing shape and body size. Future behavioral work will continue growing our bat/moth behavioral interaction datasets needed to test for and quantify the selective advantage of putative anti-bat traits.</p>

<figure class="pull-right">
	<a href="https://thehamiltonlab.github.io/images/Fig2_A.jpg">&lt;img src="https://thehamiltonlab.github.io/images/Fig2_A.jpg" alt="image" width="150"</a>
</figure>

<figure class="pull-right">
	<a href="https://thehamiltonlab.github.io/images/Science_Advances.jpg">&lt;img src="https://thehamiltonlab.github.io/images/Science_Advances.jpg" alt="image" width="150"</a>
</figure>

<h3 id="convergent-evolution">Convergent Evolution</h3>

<p>Convergence is a striking example of the power of natural selection. But is convergence a common occurrence? Evidence is mounting that it is, yet the rules that determine the patterns of convergence we see are not well understood. It is important for us to ask why convergence occurs in some cases and not others. If we rerun the “tape of life”, will the results be the same?
To do this, my research combines phylogenetics, functional genomics, geometric morphometrics, and behavioral experiments to investigate the mechanisms behind these patterns. For example, future work will look to understand whether convergence is correlated with niche and/or predatory variables (e.g., community composition, distributions, interactions), and/or genomic changes (i.e., gene expression and regulation patterns) - Can we predict where hindwing tails will evolve, or putatively have evolved, in the Saturniidae moths?</p>

<figure class="pull-right">
	<a href="https://thehamiltonlab.github.io/images/phylomorphospace3d.gif">&lt;img src="https://thehamiltonlab.github.io/images/phylomorphospace3d.gif" alt="image" width="150"</a>
</figure>

<h3 id="machine-learning">Machine Learning</h3>

<p>Current rates of species loss exceed the historical extinction rate by several orders of magnitude. This loss has been shown to destabilize Earth’s ecosystems and diminish the benefits they provide to humanity. After ~300 years of taxonomy, ~1.5 million species have been described (of the 8-9 million estimated non-microbial species). Furthermore, 57% of the 150 most commonly prescribed drugs in the U.S. contain active ingredients derived from natural compounds extracted from this biodiversity. If radical changes are not implemented, we will continue to lose species before knowing anything about their role in ecosystems or potential for human health.
One major challenge for biodiversity discovery is the time and effort required to determine and describe new species. This has led to an incredible “taxonomic backlog”. Our research team (the Hamilton Lab and collaborators) is developing machine learning tools to more easily and cheaply accelerate biodiversity discovery.</p>

<figure class="pull-right">
	<a href="https://thehamiltonlab.github.io/images/skynet2.png">&lt;img src="https://thehamiltonlab.github.io/images/skynet2.png" alt="image" width="150"</a>
</figure>

<h3 id="outreach">Outreach</h3>

<p>As a member of the Chickasaw Nation of Oklahoma, I have a personal investment to engage and mentor fellow Native American students. The inclusion of underrepresented groups in biology, is essential to enhancing scientific literacy in the United States. I recently created a program involving middle school and high school students from the Chickasaw Nation of Oklahoma to engage them into modern biological research. This program uses the exciting bat-moth evolutionary arms race to get students into nature, collect specimens, and teaches them how to participate in DNA extraction and high-throughput sequencing with an Oxford Nanopore MinION, while also learning coding. Students take the assembled sequences, identify the CO1 barcode, and use BOLD to identify the species they have collected. I am now working to adapt this approach for STEM education in Idaho, in particular our native tribes.</p>


      </div><!-- /.entry-content -->
    </div><!-- /.entry-wrapper -->
  </article>
</div><!-- /#main -->

<div class="footer-wrapper">
  <footer role="contentinfo" class="entry-wrapper">
    <footer id="footer" class="site-footer">
  <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
<div class="copyright">
    
      <p>&copy; 2019 The Hamilton Lab. Powered by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://github.com/mmistakes/so-simple-theme" rel="nofollow">So Simple</a>.</p>
    
  </div>
</footer>

  </footer>
</div><!-- /.footer-wrapper -->

<script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
  <script src="/hamilton-lab/assets/js/main.min.js"></script>
  <script src="https://use.fontawesome.com/releases/v5.0.12/js/all.js"></script>



</body>
</html>
